src/main/java/frc/robot/RobotContainer.java

```diff
@@ -186,15 +213,63 @@ private void configureBindings() {
         // Engage        
         controller.rightBumper().onTrue(CommandUtil.logged(new EngageRatchet(climber)));
 
-//        controller.a().onTrue(new StartFeeder(feeder));
-//        controller.b().onTrue(new ExitAndShoot(shooter,feeder));
-//        ramp.setDefaultCommand(new RampMove(ramp, 10));
-//        climber.setDefaultCommand(new ManualClimb(climber, controller::getLeftX));
-        controller.a().onTrue(CommandUtil.logged(new StartIntakeAndFeeder(feeder,intakeSubsystem,deployer,ramp)));
-        controller.b().onTrue(CommandUtil.logged(new ExitAndShoot(shooter,feeder, drivetrain)));
-        controller.x().onTrue(CommandUtil.logged(new LowerDeployer(deployer)));
-//        controller.a().onTrue(new DeployerLower(deployer));
-//        controller.b().onTrue(new DeployerRaise(deployer));
+        // Set up to shoot Speaker CLOSE - Y
+        controller.y().onTrue(CommandUtil.parallel("Setup Speaker Shot (CLOSE)",
+                CommandUtil.logged(new RampMove(ramp, () -> GameConstants.RAMP_POS_SHOOT_SPEAKER_CLOSE)),
```

Comment:
```suggestion
                new RampMove(ramp, () -> GameConstants.RAMP_POS_SHOOT_SPEAKER_CLOSE),
```
`CommandUtil.logged` is automatically applied

Reply:
done